fincaguarumo/src/components/icons/index.ts

```
@@ -31,8 +31,10 @@ import Waze from "./Waze"
 import Whatsapp from "./Whatsapp"
 import Youtube from "./Youtube"
 import ExternalLink from "./ExternalLink"
+import Success from "./Success"
+import Error from "./Error"
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)